CMakeLists.txt

```diff
@@ -1,18 +1,32 @@
 cmake_minimum_required(VERSION 3.5.1)
-project(CMaizeSuperbuild VERSION 1.0.0 LANGUAGES CXX)
-
-include(ExternalProject) # For ExternalProject_Add command
 
 # This is where we will install artifacts during the build stage
-set(stage_directory "${CMAKE_BINARY_DIR}/stage/${CMAKE_INSTALL_PREFIX}")
+# TODO: change stage_dir to CMAKE_INSTALL_DIR
+set(stage_directory "${CMAKE_BINARY_DIR}/stage")
+
+set(CMAKE_BUILD_TYPE_INIT "Release")
+
+project(CMaizeSuperbuild VERSION 1.0.0 LANGUAGES CXX)
 
 # Builds the main library and installs it to the staging directory
-ExternalProject_Add(
-    "${PROJECT_NAME}_core"
-    SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/hello_world"
-    CMAKE_ARGS -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
-	       -DCMAKE_INSTALL_PREFIX=${stage_directory}
-    BUILD_ALWAYS 1
+# ExternalProject_Add(
+#     "${PROJECT_NAME}_core"
+#     SOURCE_DIR "${CMAKE_CURRENT_LIST_DIR}/hello_world"
+#     CMAKE_ARGS -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+# 	       -DCMAKE_INSTALL_PREFIX=${stage_directory}
+#     BUILD_ALWAYS 1
+# )
```

Comment:
If you commented this out for testing purposes that's fine, but you're going to need this (and it's going to need to come after the `ExternalProject_Add` call for CMakePublic)